### ISSUE TITLE
Import alias

### ISSUE THREAD

**Issue by durandj** [atlassian/stricter#82]
I'm currently trying to set up stricter for a project I work on and I've been seeing issues with the `unused-files` rule that is built in. Namely, it seems as if stricter doesn't pick up on aliases that are used in imports. For example, say you have the following in your `tsconfig.json`

```json
{
  "compilerOptions": {
    "baseUrl": "./",
    "paths": {
      "myProject/*": ["src/*"],
    }
  }
}
```

and something similar in your `webpack.config.js`

```javascript
module.exports = {
  resolve: {
    alias: {
      myProject: path.resolve(__dirname, "src"),
    },
  },
};
```

If you then had an import like this it wouldn't get picked up: `import myFunction from "myProject/myFunction";`. Am I missing something or is this just not supported at this point?

**Comment by Li0liQ** [comment]
The mention of `tsconfig.json` suggests, you might be using TypeScript, which is not yet supported by `stricter` (work in progress).
However, if you are talking about javascript files, `stricter` uses [node resolution algorithm](https://nodejs.org/api/modules.html#modules_all_together) behind the scenes.
You have two options here:
- add `src` to `NODE_PATH` environment variable (e.g. `NODE_PATH=$NODE_PATH:$PWD/src; stricter`)
- migrate to using packages in a mono-repo fashion and change "src" imports into package imports

**Comment by durandj** [comment]
We are using Typescript :(

Is there someplace I can track the progress and contribute to help move
this along? Would really like to get this working.

**Comment by Li0liQ** [comment]
The latest version should support TypeScript now.
Could you, please, give it a try?

**Comment by durandj** [comment]
:open_mouth: I'll definitely try it out.

**Comment by Li0liQ** [comment]
Sorry, found an issue with it #86.
I'll keep you posted.

**Comment by durandj** [comment]
Ah that would explain the issues I was having. Was trying to figure out why some things were still not marked correctly. Thanks for the update :)

**Comment by Li0liQ** [comment]
Fixed in [0.4.3](https://github.com/atlassian/stricter/releases/tag/0.4.3).
